### PR TITLE
feat: onboarding table to prometheus

### PR DESCRIPTION
**Pull request by sjaanus**
Instead of lifecycle table, we take the metrics directly from onboarding tables.

### CODE REVIEW

File: src/lib/features/onboarding/onboarding-read-model.ts [hunk 1]
```diff
@@ -4,14 +4,19 @@ import type {
...
 } from './onboarding-read-model-type';
-import { millisecondsToMinutes } from 'date-fns';
 
-const calculateTimeDifferenceInMinutes = (date1?: Date, date2?: Date) => {
-    if (date1 && date2) {
-        const diffInMilliseconds = date2.getTime() - date1.getTime();
-        return millisecondsToMinutes(diffInMilliseconds);
-    }
-    return null;
+const instanceEventLookup = {
```
Comment (sjaanus) [inline src/lib/features/onboarding/onboarding-read-model.ts:8]: Need mapping for new column format

Comment (kwasniew) [reply src/lib/features/onboarding/onboarding-read-model.ts:8]: maybe we can use the - notation for the metrics so that we can skip mapping?

Comment (kwasniew) [reply src/lib/features/onboarding/onboarding-read-model.ts:8]: did we add camel case to be consistent with other metrics?

**Review by kwasniew — APPROVED** [review]

File: src/lib/features/onboarding/onboarding-read-model.ts (hunk 1 above)
Comment (sjaanus) [reply src/lib/features/onboarding/onboarding-read-model.ts:8]: I just dont want to mess with json object with - in the key.

**Pull request merged**